build.go

```diff
@@ -119,25 +81,18 @@ func Build(dependencyService DependencyService, cacheMatcher CacheMatcher, insta
 			if err != nil {
 				return packit.BuildResult{}, err
 			}
-
 			err = os.Symlink(filepath.Join(modulesLayer.Path, "node_modules"), filepath.Join(context.WorkingDir, "node_modules"))
 			if err != nil {
+				// not tested
```

Comment:
Is this comment left here intentionally?

Reply:
Yeah! We couldn't figure out an easy way to hit the `if` case in our test, and in those circumstances I like to be transparent.